pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -78,20 +99,55 @@ func makeSimpleImportSpans(
 
 		for layer := range backups {
 			covPos := spanCoverStart
+			var curCovGrowSize int64
```

Comment:
maybe add def `curGovGrowSize`? I think it's: curCovGrowSize is the number of bytes added to the right most cover since its creation.